model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -392,15 +422,20 @@ case object Dockstore extends MethodRepository {
   override val scheme: String = "dockstore"
 }
 
+case object DockstoreTools extends MethodRepository {
+  override val scheme: String = "dockstoretools"
+}
+
 object MethodRepository {
 
   def withName(name: String): Option[MethodRepository] = name.toLowerCase match {
     case Agora.scheme => Some(Agora)
     case Dockstore.scheme => Some(Dockstore)
+    case DockstoreTools.scheme => Some(DockstoreTools)
     case _ => None
   }
 
-  val all: Set[MethodRepository] = Set(Agora, Dockstore)
+  val all: Set[MethodRepository] = Set(Agora, Dockstore, DockstoreTools)
```

Comment:
It would be nice to throw in a comment, perhaps here, explaining the domain concept difference between `Dockstore` and `DockstoreTools`

Reply:
👍